app/controllers/rubcop_test_controller.rb

```diff
@@ -0,0 +1,17 @@
+class RubocopController  < ApplicationController
+	def initialize
+		print "this is so bad patern"
```

Comment:
__[staging][rubocop]__
* Tab detected.
* Use 2 (not 1) spaces for indentation.
* Prefer single-quoted strings when you don't need string interpolation or special symbols.